app/app-upgrade-client/src/dashboard.tsx

```diff
@@ -45,6 +48,6 @@ export const Dashboard: FC = () => {
           <OnboardingForm />
         </div>
       </Modal>
-    </>
+    </React.Fragment>
   );
 };
```

Comment:
We should rename the APP_UPGRADE_white to app-upgrade-logo-white and then use an svg.

Reply:
done